File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Responses/RedeemPromotionsResponse.cs
using Olo.Promotions.SDK.Responses.Models;

namespace Olo.Promotions.SDK.Responses
{
    /// <summary>
    /// Returned for a successful call to the Redeem Promotions endpoint.
    /// </summary>
    public class RedeemPromotionsResponse
    {
        public TransactionWithPromotions Transaction { get; set; }
    }
}

File: olo-promotions-sdk-go/pkg/promotions/models/responses/promotions/redeem_promotions_response.go
package promotions

import "olo-promotions-sdk-go/pkg/promotions/models/responses"

// RedeemPromotionsResponse is returned for a successful call to the Redeem Promotions endpoint.
type RedeemPromotionsResponse struct {
	// Transaction contains the transaction details along with applied promotions.
	Transaction responses.TransactionWithPromotions `json:"transaction"`
}
